src/operations/execute_operation.ts

```diff
@@ -246,21 +248,20 @@ function executeWithServerSelection(
   }
 
   // select a server, and execute the operation against it
-  topology.selectServer(selector, serverSelectionOptions, (err?: any, server?: any) => {
-    if (err) {
-      callback(err);
-      return;
+  topology.selectServer(selector, serverSelectionOptions, (error, server) => {
+    if (error || !server) {
+      return callback(error);
     }
 
     if (session && operation.hasAspect(Aspect.RETRYABLE)) {
       const willRetryRead =
-        topology.s.options.retryReads !== false &&
+        topology.s.options.retryReads !== false && // why is this not false
```

Comment:
the world wants to know; they both get set to true by default, and they are both supposed to be boolean as defined on the MongoClient, so I say we should update both places to match; in fact, we should probably just use `.retryReads` (no `===` or `!==`)